modules/aws-emr-ports/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=4.0.0"
+    }
+  }
```

Comment:
The ports module doesn't actually require aws.

It's essentially just a keeper of lists of port numbers. No real resources required.